pkg/controller/master/setting/additional_ca.go

```diff
@@ -0,0 +1,63 @@
+package setting
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	harvesterv1 "github.com/harvester/harvester/pkg/apis/harvesterhci.io/v1beta1"
+	"github.com/harvester/harvester/pkg/util"
+)
+
+func (h *Handler) syncAdditionalTrustedCAs(setting *harvesterv1.Setting) error {
+	// Add envs to the backup secret used by Longhorn backups
+	backupConfig := map[string]string{
+		"AWS_CERT": setting.Value,
+	}
+	if err := h.updateBackupSecret(backupConfig); err != nil {
+		return err
+	}
+
+	// Distribute CA secrets to required system namespaces
+	if err := h.syncAdditionalCASecrets(setting); err != nil {
+		return err
+	}
+
+	//redeploy system services. The CA certs will be injected by the mutation webhook.
+	return h.redeployDeployment(h.namespace, "harvester")
```

Comment:
@gitlawr  The mutation hook also checks pods with the label `"longhorn.io/component": "backing-image-data-source"`.
Does anything else need to be restarted here?

Reply:
They are ephemeral like job pods